assets/i18n/en.json

```diff
@@ -778,6 +781,7 @@
     "I18N_TOPIC_VIEWER_SKILLS": "Skills",
     "I18N_TOPIC_VIEWER_START_PRACTICE": "Start",
     "I18N_TOPIC_VIEWER_STORIES": "Stories",
+    "I18N_TOPIC_VIEWER_STORIES_HEAD": "Stories You Can Play",
```

Comment:
Let it be `Stories you can play`, and use css property `text-transform:capitalize` so that in other languages while translating it doesn't create a problem.

Reply:
Done!